tox.ini

```diff
@@ -6,6 +6,35 @@
 
 [testenv]
 deps=-rtests/requirements.txt
-passenv=APPDATA USERPROFILE HOMEDRIVE HOMEPATH windir APPVEYOR
-commands=py.test {posargs}
+setenv=COVERAGE_FILE={toxworkdir}/.coverage.{envname}
+passenv=APPDATA USERPROFILE HOMEDRIVE HOMEPATH windir APPVEYOR APPVEYOR_* CI CODECOV_* TRAVIS TRAVIS_*
+# These are separate so xfail tests don't count towards code coverage
+commands=pytest -m "not xfail" -k test_dist {posargs: "{toxinidir}/setuptools/tests" --cov-config={toxinidir}/tox.ini --cov=setuptools}
```

Comment:
Is only testing `test_dist` intended?

Reply:
`pkg_resources` is also not tested anymore.

Reply:
Nope, oops.

Reply:
I did that to make the CI runs faster and meant to rebase delete it. That explains the super low code coverage metrics :P